.github/workflows/deploy-af-mvp.yml

```diff
@@ -49,3 +49,10 @@ jobs:
           command: up
           stack-name: ${{ env.pulumi_stack_organization }}/${{ inputs.environment }}
           upsert: true # create stack if no exists
+      - name: Initial deployment domain check
+        if: steps.pulumi.outputs.initialDomainCheckRequired
+        uses: pulumi/actions@v4
+        with:
+          work-dir: ./infra/af-mvp
+          command: up
+          stack-name: ${{ env.pulumi_stack_organization }}/${{ inputs.environment }}
```

Comment:
Olisko tässä hyvä oikein kädestä pitäen kertoa, että tämä viimeinen steppi tarkastelee edellistä (kyseiselle stepille voisi vielä vaikka antaa id:ksi `pulumi`, niin vscode ei herjaa `Context access might be invalid: pulumi`). Ehkä astetta luettavampi ja ei aiheuta IDE warningien kanssa hämmennystä, että mihin `steps.pulumi` viittaa. Ehkä jopa viekä kommenttiin, että tuo deplaus tosiaan sylkäisee tuon `initialDomainCheckRequired` outputeistaan.

Reply:
Jahas, ei ollut vscodessa GHA-plugari päällä. Steppi saa jo valmiiksi ID:n "pulumi" koska kyseinen action antaa sille sellaisen tunnisteen, mutta eipä kieltämättä IDE tälläisestä tiedä.

Reply:
Jeps, näin laskin kanssa 1 + 1 että sieltä tulee, mutta IDE ei ole vielä tarpeeksi viisas.

Reply:
Tämän varoituksen varmaan voisi ehjätä siten että tekisi oman toteutuksen tuosta `pulumi/actions@v4`-toimesta jonka kautta outputit ja vastaavat tulisi gha-vscoden näkyville, mutta muuten vähän näyttää että kyseessä on ehkä virhe itse tuossa plugarissa. 

Helppo ratkaisu olisi tässä kommentissa: https://github.com/github/vscode-github-actions/issues/67#issuecomment-1645030256
Valittu ratkaisu, eli purkkafiksi, olisi taas tässä: https://github.com/github/vscode-github-actions/issues/222#issuecomment-1732200017